tests/interpreter/cases/input/mod.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#![cfg(test)]
+
+use crate::interpreter::*;
+
+#[test]
+fn basic_second_input() -> Result<()> {
+    let rego = r#"
+    package test
+
+    x[a] {
+        a = input.x + 5
+    }
+"#;
+
+    let expected = Value::from_json_str(
+        r#" {
+            "test": {
+                "x": {
+                    "set!":[8, 15]
```

Comment:
@anakrish result should only be "set!":[15] and not "set!":[8, 15]? We should clean the data on each evaluation?

[With OPA](https://pkg.go.dev/github.com/open-policy-agent/opa/rego#Rego.PrepareForEval) For example, in OPA, results are independent of each other on different evaluations.